baseclasses/BaseRegTest.py

```diff
@@ -58,13 +58,6 @@ def __init__(self, ref_file, train=False, comm=None, check_arch=False):
             assert os.path.isfile(self.ref_file)
             self.db = self.readRef()
 
-        # dictionary of real/complex PETSc arch names
```

Comment:
Why is this not necessary anymore?

Reply:
We never really used this, which were ported over from the old regression tests so that you can run real/complex tests in pyAeroStructure in a single python script. Now we just export `$PETSC_ARCH` and test real/complex separately. There may be _some_ value in this but IMO we can choose to resurrect this if needed. I don't like having dead code lying around that's not used or tested.